src/bun.js/bindings/ZigGlobalObject.cpp

```diff
@@ -1779,6 +1784,89 @@ JSC_DEFINE_HOST_FUNCTION(jsReceiveMessageOnPort, (JSGlobalObject * lexicalGlobal
     return JSC::JSValue::encode(jsUndefined());
 }
 
+JSC_DEFINE_HOST_FUNCTION(functionDomainToASCII, (JSC::JSGlobalObject * globalObject, JSC::CallFrame* callFrame))
+{
+    auto& vm = globalObject->vm();
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    if (callFrame->argumentCount() != 1) {
```

Comment:
The convention in JavaScript is not to error on additional arguments. The convention is to error if insufficient arguments.

```suggestion
    if (callFrame->argumentCount() < 1) {
```